### ISSUE TITLE
Imposible to use many sliders in the same time on a device with touch screen (by multitouch)

### ISSUE THREAD

**Issue by daniela-mateescu** [angular-slider/angularjs-slider#534]
### Steps to reproduce
1. Having a device with a touch screen
2. Having two sliders on the same screen
3. Start dragging first slider 
4. Start dragging second slider in the same time using another finger
5. Continue dragging both sliders in the same time
Demo: http://jsfiddle.net/1o71k7j5/1/

### Expected behaviour
Both sliders are dragged correctly, according to the corresponding finger that initiated the drag

### Actual behaviour
Only the second touched slider is moving when moving both fingers. Also this slider moves incorrectly because it follows the movement of the finger that initiated the drag on the other slider.

### Notes
We already done some internal test by extending your component and adding the fix to the above issue. We want to create a pull request for this change. But before starting to integrate this fix on your library we would like to know if you think that it would be ok to integrate it? 

Thank you and looking forward for your response.

**Comment by ValentinH** [comment]
Hello. Any pull requests are always welcome. The only conditions are:
- it does not break other features
- it keeps the test coverage at 100%

Thus, I would be more than happy to provide any useful information if you need. ;)
